modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -7,32 +7,17 @@
 [id="installation-user-infra-machines-static-network_{context}"]
 = Advanced {op-system} installation reference
 
-This section illustrates the networking configuration and other advanced options that allow you to modify the {op-system-first} bare metal install process. The following tables describe the kernel arguments and command-line options you can use with:
-
-* The {op-system} live installer boot prompt by using routing and bonding.
-
-* The {op-system} live installer by using `core.inst` for ISO or PXE installs.
-
-* The `coreos-installer` command for ISO installs.
-
-[discrete]
-== Routing and bonding options at {op-system} boot prompt
-
-If you install {op-system} from an ISO image, you can add kernel arguments manually when you boot that image to configure the node's networking. If no networking arguments are used, the installation defaults to using DHCP.
-
-[IMPORTANT]
-====
-When adding networking arguments, you must also add the `rd.neednet=1` kernel argument.
-====
-
-The following table describes how to use `ip=`, `nameserver=`, and `bond=` kernel arguments for live ISO installs.
+If you install {op-system-first} from an ISO image, you can add kernel arguments
+when you boot that image to configure the node's networking.
+The following table describes and illustrates how to use those kernel arguments.
```

Comment:
As noted in https://github.com/openshift/openshift-docs/pull/26559, I don't see why this text was changed. The original wording introduces the three options and tables we provide for advanced networking config in this section.